include/drm/drm_atomic.h

```diff
@@ -428,6 +428,7 @@ struct drm_atomic_state {
 	 * commit without blocking.
 	 */
 	struct work_struct commit_work;
+	int restore;
```

Comment:
it is better change to:
/* indicate if system just resume from S3, will used it to control if print first commit period */
bool resumed : 0;